doc-site/docs/concepts/tokens.md

```diff
@@ -0,0 +1,104 @@
+![Transformation of Tokens](../images/paladin_tokens.svg)
+
+Tokens are the foundation on which blockchain and DLT use cases are built.
+
+A token is a digital record of the existence and ownership of something of value.
+
+The ability to refer to and transfer ownership of a token as part of an _atomic transaction_, is what makes Web3 (Blockchain and DLT) technologies fundamentally different to the API driven world of Web2.
+
+## Tokens standards on EVM
+
+EVM has established really important standards for interoperability of tokens.
+
+- ERC-20 and related standards like ERC-777
+    - These provide an interoperability standard for `fungible` tokens, where value is divisible and interchangeable within the pool of value represented by the token
+- ERC-721 and related standards like ERC-1155
+    - These provide an interoperability standard for `non-fungible` tokens, representing a unique item with a unique identity
+    - In the case of ERC-1155 the ability to blur the lines between fungible and non-fungible use cases
+- ERC-1400 (ERC-14xx), ERC-3643 (T-Rex) and related standards
+    - These provide various levels of reference and standardization for the more complex realities of business use cases
+    - Implementors might use these as an accelerator for building real-world enterprise uses cases using tokens, although none provide a one-size-fits-all solution
+    - Generally tokens still conform to ERC-20/ERC-721/ERC-1155 when encompassing more complex standards, as that is where the most interoperability and programmability comes from
+
+> None of these standards address privacy, because these tokens requires complete visibility into the transaction history to trust the value.
+
+## The privacy requirements
+
+To build enterprise use cases meaningfully that use tokens, the data must be segregated in a way that allows each party to see and be in control of an appropriate subset of data in the transaction.
+
+### The token itself
+
+The token itself needs to have a wide reach, to be a provably unique record of ownership across the whole ledger/blockchain. This is what makes a token so powerful.
+
+In a privacy preserving use case, it is necessary to avoid the history of ownership of a token being directly or indirectly available to those with access to the chain. However, it must still be true that when I receive a token that I can trust the legitimacy of the toke within the scope of the ledger.
```

Comment:
```suggestion
In a privacy preserving use case, it is necessary to avoid the history of ownership of a token being directly or indirectly available to those with access to the chain. However, it must still be true that when I receive a token that I can trust the legitimacy of the token within the scope of the ledger.
```